tests/tests.rs

```diff
@@ -64,10 +78,32 @@ fn test_simple() {
     );
 }
 
+/// Test multiple directory searches
+#[test]
+fn test_multi_file() {
+    let dirs = &["test1", "test2"];
+    let files = &["test1/a.foo", "test1/b.foo", "test2/a.foo"];
+    let te = TestEnv::new(dirs, files);
+    te.assert_output(
```

Comment:
Maybe we could also add a test for `["a.foo", "test1"]` here to make sure that `test2/a.foo` is *not* found in this case.

Reply:
Yea that certainly can be done, I'll add that tonight. If you think of any other tests that should be included let me know.

Reply:
No, the rest looks great